base.gradle

```
@@ -149,6 +149,16 @@ android {
             }
         }
     }
+
+    flavorDimensions "license"
+    productFlavors {
+        free {
+            dimension "license"
+        }
+        gplay {
+            dimension "license"
```

Let's make the `gplay` dimension the default one.

```suggestion
            dimension "license"
            isDefault true
```